hummingbot/core/web_assistant/connections/ws_connection.py

```diff
@@ -4,11 +4,14 @@
 from typing import Any, Dict, Mapping, Optional
 
 import aiohttp
+from aiohttp import WebSocketError, WSCloseCode
 
 from hummingbot.core.web_assistant.connections.data_types import WSRequest, WSResponse
 
 
 class WSConnection:
+    _MAX_MSG_SIZE = 4 * 1024 * 1024  # default aiohttp: 4 * 1024 * 1024
+
     def __init__(self, aiohttp_client_session: aiohttp.ClientSession):
```

Comment:
Did you add this? I'm missing it in my system.

Here is how it shows in my system

class WSConnection:
self._client_session = aiohttp_client_session
self._connection: Optional[aiohttp.ClientWebSocketResponse] = None
self._connected = False
self._message_timeout: Optional[float] = None
self._last_recv_time = 0

cc @isreallee82 @MementoRC